packages/flutter/lib/src/material/colors.dart

```diff
@@ -333,11 +332,26 @@ class Colors {
   ///  * [ExpandIcon], which uses this color for dark themes.
   ///  * [Theme.of], which allows you to select colors from the current theme
   ///    rather than hard-coding colors in your build methods.
-  ///  * [white, white30, white12, white10], which are variants on this color
+  ///  * [white, white50, white30, white12, white10], which are variants on this color
   ///    but with different opacities.
   static const Color white54 = Color(0x8AFFFFFF);
 
-  /// White with 32% opacity.
+  /// White with 50% opacity.
+  ///
+  /// Used for disabled icons in dark themes.
```

Comment:
@HansMuller I cannot tell if disabledColor should be at 50% or 32% opacity for dark themes, since there are two conflicting sources of truth:
- [https://material.io/design/iconography/system-icons.html#color](MD Icon color) vs [https://material.io/design/color/dark-theme.html#ui-application](MD Dark theme)

Right now I'm thinking it might be better to stick with the dark theme recommendation?

Reply:
Sounds good to me, I'll make this a separate PR and revert the values for now.

Reply:
Tracking here: https://github.com/flutter/flutter/issues/33149